docs/IndicatorSize.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I think the case is because we use custom propType checker instead. Since I'm gonna revert it, this doc is will be better left out. Thanks!